src/main/java/com/gtnewhorizons/angelica/mixins/early/angelica/vbo/MixinRenderGlobal.java

```diff
@@ -38,15 +37,12 @@ private int generateDisplayLists(int range) {
         return VBOManager.generateDisplayLists(range);
     }
 
-    @Override
     @Redirect(method="<init>", at = @At(value="INVOKE", target="Lorg/lwjgl/opengl/GL11;glNewList(II)V", ordinal = 0), remap = false)
-    public void startStarsVBO(int list, int mode) {
+    public void angelica$startStarsVBO(int list, int mode) {
```

Comment:
Mixin generates unique names for redirector methods, so the prefix is unnecessary (and is inconsistent with the rest of the methods anyway).

Reply:
hah, I originally took the same approach as you and then decided that was silly and didn't put things back properly